modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ExportImportObjectDefinitionTest.java

```diff
@@ -166,6 +149,58 @@ private MockLiferayResourceRequest _createMockLiferayResourceRequest(
 		return mockMultipartHttpServletRequest;
 	}
 
+	private void _exportImportObjectDefinition(
+			String fileName, String name, Boolean system)
```

Comment:
@pedro-oliveira446 use boolean instead of the class Boolean, we don't need to use the class in this case